exchanges/gateio/gateio_wrapper.go

```diff
@@ -710,7 +727,39 @@ func (g *Gateio) ValidateCredentials() error {
 	return g.CheckTransientError(err)
 }
 
+// FormatExchangeKlineInterval returns Interval to exchange formatted string
+func (g *Gateio) FormatExchangeKlineInterval(in kline.Interval) string {
+	return strconv.FormatFloat(in.Duration().Seconds(), 'f', 0, 64)
+}
+
 // GetHistoricCandles returns candles between a time period for a set time interval
-func (g *Gateio) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval time.Duration) (kline.Item, error) {
-	return kline.Item{}, common.ErrNotYetImplemented
+func (g *Gateio) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
+	if !g.KlineIntervalEnabled(interval) {
+		return kline.Item{}, kline.ErrorKline{
+			Interval: interval,
+		}
+	}
+
+	hours := end.Sub(start).Hours()
+	params := KlinesRequestParams{
+		Symbol:   pair.String(),
```

Comment:
The pair needs to be formatted before this point (I'm not sure where). Calling `./gctcli.exe gethistoriccandlesextended gateio BTC-USDT spot --g=60 --r=10` does not return data and the params are: `{Symbol:BTC-USDT HourSize:720 GroupSec:60}`. However, using gateio test, the currency needs to be `{Symbol:BTC_USDT HourSize:720 GroupSec:60}` to return data